Dockerfile

```diff
@@ -1,13 +1,19 @@
-FROM debian:stretch
+FROM debian:testing
 
 RUN set -e \
- && apt-get update \
- && apt-get install -y procps curl rsync kmod ssh \
+ && apt-get update -yqq \
+ && apt-get install -yqq git procps curl rsync kmod ssh ruby ruby-dev build-essential \
```

Comment:
So the image is twice as large now. I don't like it, but if there is no simple way to reduce it again, at least make an issue pls.

Reply:
One option could be to have two "from"s in the build, and copy over the entire `/opt` folder, after installing vagrant and the plugin.

Reply:
The plugins are hidden away in `/root/.vagrant.d/...`. Sure, [here's an issue](https://github.com/vmck/vagrant-vmck/issues/10).